system-parachains/people/people-polkadot/bridge-primitives/src/lib.rs

```diff
@@ -0,0 +1,94 @@
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Module with configuration which reflects PeopleHubPolkadot runtime setup
+//! (AccountId, Headers, Hashes...)
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+
+pub use bp_bridge_hub_cumulus::*;
+use bp_messages::*;
+use bp_runtime::{
+	decl_bridge_finality_runtime_apis, decl_bridge_messages_runtime_apis, Chain, ChainId, Parachain,
+};
+use frame_support::dispatch::DispatchClass;
+use sp_runtime::{RuntimeDebug, StateVersion};
+
+/// PeopleHubPolkadot parachain.
+#[derive(RuntimeDebug)]
+pub struct PeopleHubPolkadot;
+
+impl Chain for PeopleHubPolkadot {
+	const ID: ChainId = *b"phpd";
+	const STATE_VERSION: StateVersion = StateVersion::V1;
+
+	type BlockNumber = BlockNumber;
+	type Hash = Hash;
+	type Hasher = Hasher;
+	type Header = Header;
+
+	type AccountId = AccountId;
+	type Balance = Balance;
+	type Nonce = Nonce;
+	type Signature = Signature;
+
+	fn max_extrinsic_size() -> u32 {
+		*BlockLength::get().max.get(DispatchClass::Normal)
+	}
+
+	fn max_extrinsic_weight() -> Weight {
+		BlockWeights::get()
+			.get(DispatchClass::Normal)
+			.max_extrinsic
+			.unwrap_or(Weight::MAX)
+	}
+}
+
+impl Parachain for PeopleHubPolkadot {
+	const PARACHAIN_ID: u32 = PEOPLE_HUB_POLKADOT_PARACHAIN_ID;
+	const MAX_HEADER_SIZE: u32 = MAX_BRIDGE_HUB_HEADER_SIZE;
+}
+
+impl ChainWithMessages for PeopleHubPolkadot {
+	const WITH_CHAIN_MESSAGES_PALLET_NAME: &'static str =
+		WITH_PEOPLE_HUB_POLKADOT_MESSAGES_PALLET_NAME;
+	const MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX: MessageNonce =
+		MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+	/// This constant limits the maximum number of messages in `receive_messages_proof`.
+	/// We need to adjust it from 4096 to 2024 due to the actual weights identified by
+	/// `check_message_lane_weights`. A higher value can be set once we switch
+	/// `max_extrinsic_weight` to `BlockWeightsForAsyncBacking`.
+	const MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX: MessageNonce = 2024;
+}
+
+/// Identifier of PeopleHubPolkadot in the Polkadot relay chain.
+pub const PEOPLE_HUB_POLKADOT_PARACHAIN_ID: u32 = 1004;
+
+/// Name of the With-PeopleHubPolkadot messages pallet instance that is deployed at bridged chains.
+pub const WITH_PEOPLE_HUB_POLKADOT_MESSAGES_PALLET_NAME: &str = "PeoplePolkadotMessages";
+
+/// Name of the With-PeopleHubPolkadot bridge-relayers pallet instance that is deployed at bridged
+/// chains.
+pub const WITH_PEOPLE_HUB_POLKADOT_RELAYERS_PALLET_NAME: &str = "PeopleRelayers";
+
+/// Pallet index of `BridgePolkadotBulletinMessages: pallet_bridge_messages::<Instance1>`.
+pub const WITH_PEOPLE_POLKADOT_TO_BULLETIN_MESSAGES_PALLET_INDEX: u8 = 61;
+
+
+decl_bridge_finality_runtime_apis!(people_hub_polkadot);
```

Comment:
Please, clean this and change all the constants to people_polkadot instead of people_hub (to avoid confusion, because we don't have "People Hub", I got it, but others might start to ask question what is "People Hub")
```suggestion
decl_bridge_finality_runtime_apis!(people_polkadot);
```